clients/ctl/admin-ui/src/features/config-wizard/SuccessPage.tsx

```diff
@@ -16,24 +16,20 @@ import {
 } from "@fidesui/react";
 import React from "react";
 
-import { useAppDispatch, useAppSelector } from "~/app/hooks";
 import { StepperCircleCheckmarkIcon } from "~/features/common/Icon";
 import {
   useGetAllSystemsQuery,
   useGetSystemByFidesKeyQuery,
 } from "~/features/system/system.slice";
+import { System } from "~/types/api";
 
-import {
-  changeReviewStep,
-  changeStep,
-  selectSystemFidesKey,
-} from "./config-wizard.slice";
-
-const SuccessPage = () => {
-  const systemFidesKey = useAppSelector(selectSystemFidesKey);
-  const dispatch = useAppDispatch();
-
-  const { data: existingSystem } = useGetSystemByFidesKeyQuery(systemFidesKey);
+interface Props {
+  systemKey: System["fides_key"];
+  onContinue: () => void;
+  onAddNextSystem: () => void;
+}
+const SuccessPage = ({ systemKey, onAddNextSystem, onContinue }: Props) => {
```

Comment:
I guess it's a bit unfortunate this is called "Page" even though it's not a Next page.

Reply:
changed the name!